hack/tools/tools.go

```diff
@@ -21,7 +21,6 @@ package tools
 import (
 	_ "github.com/a8m/envsubst"
 	_ "github.com/golang/mock/mockgen"
-	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
```

Comment:
If I'm not mistaken, tools like dependabot will struggle to flag anything not in the manifest.

Do you have any insight?

Reply:
Ah, that's painful. Thanks!